### ISSUE TITLE
label

### ISSUE THREAD

**Issue by chenhuiphd** [chaimi2013/3R#7]
Hello, I downloaded the dataset folder and found that there are nighttime hazy folder and the corresponding clean images during the daytime in the folder. Could you please tell me where the nighttime label images is? Otherwise, where did your nighttime label with supervised training for haze removal come from?

**Comment by chaimi2013** [comment]
> Hello, I downloaded the dataset folder and found that there are nighttime hazy folder and the corresponding clean images during the daytime in the folder. Could you please tell me where the nighttime label images is? Otherwise, where did your nighttime label with supervised training for haze removal come from?

The ground truth should be the low-light image without color cast and haze. Note that you can use the NDR training set to train your model.
